static/sass/_snapcraft_details_heading.scss

```diff
@@ -148,4 +148,20 @@
       opacity: 0;
     }
   }
+
+  // because of Vanilla `* + *` top margin
+  .p-tooltip__message {
+    margin: 0;
+  }
+
+  // positioning fixes for track tooltip
+  .p-tooltip--track {
```

Comment:
This seems like an issue when applying a tooltip to an icon. It's unlikely any text will be smaller then tooltip icon offset. It would be nice if this could solve that issue by changing the position if on a tooltip or creating an icon related override for all the tooltip directions. 

Therefore, it would be good to have this renamed to `.is-icon` or something and create an issue on Vanilla.

Reply:
@anthonydillon Yes, it's because of using tooltip on the icon.
I updated class name to make it more explicit what it's applied to an icon. I can't make it a fully applicable Vanilla override, because adding custom `is-icon` styles also overrides every tooltip varian, when it's really needed for default (bottom left without any modifier class).

Reply:
Also added Vanilla bug: https://github.com/vanilla-framework/vanilla-framework/issues/1791